libvast/src/index/address_index.cpp

```diff
@@ -3,16 +3,16 @@
 //   | |/ / __ |_\ \  / /          Across
 //   |___/_/ |_/___/ /_/       Space and Time
 //
-// SPDX-FileCopyrightText: (c) 2020 The VAST Contributors
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
```

Comment:
The copyright date here is the file creation date, not the time it was last touched. Please revert.

Reply:
Ok. It will be reverted by next push.